### DISCUSSION TITLE
jsPsych 7.0.0

### DISCUSSION THREAD

**Discussion by rbruno1** [jspsych/jsPsych#2292] [Q&A] [answered]
Experiment not loading in version 7.0.0. The developer tool says that jsPsych is not defined.

**Comment by jodeleeuw** [comment, marked as answer]
Please check out the migration guide: https://www.jspsych.org/7.0/support/migration-v7/

**Reply by becky-gilbert** [reply]
Specifically, it sounds like you might be trying to access `jsPsych` functions before initializing with `var jsPsych = initJsPsych(...)`: https://www.jspsych.org/7.0/support/migration-v7/#initializing-and-running-jspsych
